src/references/selecting_a_storage_type.md

```diff
@@ -0,0 +1,65 @@
+---
+id: selecting_a_storage_type
+name: Selecting a Storage Type
+title: Selecting a Storage Type
+tags:
+  - Reference
+  - Tutorial
+---
+
+# Selecting a Storage Type
+
+**Core** has different storage types that can be used to save data for players, or for your game. Selecting a storage type for your game is an important design decision, but it is not always clear on the best storage type to use. To help creators pick a suitable storage type the sections below will go over various ways they can be used.
+
```

Comment:
Not sure where to put this but it would be useful to clearly say why using Concurrent Storage for something Player Storage could do is more efficient.

Reply:
I think that info could go in the reference instead. I think the main goal for this would be a quick lookup for people wondering which key type to use. What do you think?

Reply:
That's reasonable. In that case, I think the ref may be fine as is